x-pack/plugins/ml/public/application/jobs/new_job/pages/components/pick_fields_step/components/geo_view/description.tsx

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo, FC } from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { EuiDescribedFormGroup, EuiFormRow } from '@elastic/eui';
+
+export const Description: FC = memo(({ children }) => {
+  const title = i18n.translate('xpack.ml.newJob.wizard.pickFieldsStep.geoField.title', {
+    defaultMessage: 'Geo field',
+  });
+  return (
+    <EuiDescribedFormGroup
+      title={<h3>{title}</h3>}
+      description={
+        <FormattedMessage
+          id="xpack.ml.newJob.wizard.pickFieldsStep.geoField.description"
+          defaultMessage="Select a geo field. The lat_long function detects anomalies in the geographic location of the input data."
```

Comment:
```suggestion
          defaultMessage="Select a geo field to detect anomalies in the geographic location of the input data."
```

Reply:
Updated in 9be34fed1d464e8fac17090c2aa5e2e78099e150